prow/plugins/project/project.go

```diff
@@ -36,9 +36,12 @@ const (
 )
 
 var (
-	projectRegex              = regexp.MustCompile(`(?m)^/project\s+(.+?)(?:\s+(.+?)\s*)?$`)
+	projectRegex              = regexp.MustCompile(`(?m)^(?:\/(project)\s+(project=(?:'.*?'|".*?"))(?:\s+(column=(?:'.*?'|".*?"))\s*?$)?)|(?:\/(remove-project)\s+(project=(?:'.*?'|".*?"))\s*?$)`)
```

Comment:
If we don't care to enforce matching quotation marks (which seems fine to me?) we can simplify this a lot:

https://play.golang.org/p/MWNQIaywL5E

Reply:
@stevekuznetsov yeah that does seem a lot simpler. I think it's fine to allow a command like `/project project='0.0.0" column="To do'`. Thanks for the suggestion!